src/main/resources/interface-specification.yml

```diff
@@ -2134,79 +2134,6 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
-  '/institutions/{tenantId}':
```

Comment:
are we sure this endpoint is no more required?

Reply:
I think it got moved under /tenants/{tenantsId} at line 3054